src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSchema.cs

```diff
@@ -120,6 +131,10 @@ public override int GetHashCode()
 
         public static SchemaTypeName RecordTypeName => new () { Name = "Record", IsTable = false };
 
+        public static SchemaTypeName ExpandableTableTypeName => new () { Name = "ExpandableTable", IsTable = true };
```

Comment:
>ExpandableTable [](http://example.com/codeflow?start=81&length=15)

The point of `IsTable` is that that's what distinguishes table. This should just be `ExpandableRecord` with IsTable == true.

Reply:
see XR doc